### PR TITLE
Update to v8 with new packages, pinning dt2, requirements.txt for venv, helper scripts and general cleanup.

### PR DESCRIPTION
**Pull request by sei-amellinger**
- Add hjson to packages
- Added requirements.txt for venv install.
- Clarified Dockerfile pip install ordering.
- Clarified Dockerfiles w.r.t. to the scripts at the end.
- Added comments for how to pin versions.
- Added platform info tool that shows all packages and information.
- Pinned DT2 to 0.5 because 0.6 doesn't work for us yet.

### CODE REVIEW

**Comment by sei-wshaw** [comment]
I'm getting an error when I try to run pytest after building the container on gpu02:

E   ImportError: /opt/conda/lib/python3.8/site-packages/mmcv/_ext.cpython-38-x86_64-linux-gnu.so: undefined symbol: _Z27points_in_boxes_cpu_forwardN2at6TensorES0_S0_

I'm checking to see if there's something wrong with the way I built or setup the thing.

File: docker/build.sh [hunk 1]
```diff
@@ -1,12 +1,17 @@
...
 
-if [ $# -ne 1 ]; then
+if [ $# -lt 1 ]; then
   echo "This script requires one argument, the part BEFORE the '.Dockerfile'"
   echo "e.g. 'cudadev' or 'cpudev'"
   exit -1
 fi
 
-TARGET_TAG="juneberry/${1}:dev"
+REV="dev"
```
Comment (sei-wshaw) [inline docker/build.sh:9]: Should the script's usage statement reflect this new argument?

Comment (sei-amellinger) [reply docker/build.sh:9]: Sure, I can add something.

**Comment by sei-amellinger** [comment]
> I'm getting an error when I try to run pytest after building the container on gpu02:
> 
> E ImportError: /opt/conda/lib/python3.8/site-packages/mmcv/_ext.cpython-38-x86_64-linux-gnu.so: undefined symbol: _Z27points_in_boxes_cpu_forwardN2at6TensorES0_S0_
> 
> I'm checking to see if there's something wrong with the way I built or setup the thing.

Which one are you building?

**Review by sei-nwinski — APPROVED** [review]

**Pull request merged**